tests/ctst/common/common.ts

```diff
@@ -300,40 +300,58 @@ Then('i {string} be able to add user metadata to object {string}',
         }
     });
 
-Then('kafka consumed messages should not take too much place on disk',
+Then('kafka consumed messages should not take too much place on disk', { timeout: 130000 },
     async function (this: Zenko) {
         const ignoredTopics = ['dead-letter'];
         const kafkaAdmin = new Kafka({ brokers: [this.parameters.KafkaHosts] }).admin();
         const topics: string[] = (await kafkaAdmin.listTopics())
             .filter(t => (t.includes(this.parameters.InstanceID) &&
             !ignoredTopics.some(e => t.includes(e))));
-        const previousOffsets = await getTopicsOffsets(topics, kafkaAdmin);
 
-        const seconds = parseInt(this.parameters.KafkaCleanerInterval);
+        const timeStart = new Date().getTime();
 
-        // Checking topics offsets before kafkacleaner passes to be sure kafkacleaner works
-        // This function can be improved by consuming messages and
-        // verify that the timestamp is not older than last kafkacleaner run
-        // Instead of waiting for a fixed amount of time,
-        // we could also check for metrics to see last kafkacleaner run
-        
-        // 10 seconds added to be sure kafkacleaner had time to process
-        await Utils.sleep(seconds * 1000 + 10000);
+        while (topics.length > 0 && new Date().getTime() - timeStart < 120000) {
```

Comment:
what is the point of checking the duration?
* the timeout on the test is designed to catch test taking too long to complete
* here we simply want to try "a few times"

Reply:
This is to have the while leaving and at the end we can log each topic that is not cleaned in time, if we just wait for the timeout, we'll end up with no readable logs.. just "test timed out" with no info..